plugins/main/public/components/overview/fim/inventory/inventory.tsx

```diff
@@ -41,6 +41,8 @@ import { withErrorBoundary } from '../../../common/hocs';
 import { HitsCounter } from '../../../../kibana-integrations/discover/application/components/hits_counter/hits_counter';
 import { formatNumWithCommas } from '../../../../kibana-integrations/discover/application/helpers';
 import { useAppConfig } from '../../../common/hooks';
+import { WAZUH_INDEX_TYPE_FIM } from '../../../../../common/constants';
+import useCheckIndexFields from '../../vulnerabilities/common/hooks/useCheckIndexFields';
```

Comment:
Perhaps the vulnerability files should be moved if they have not already been moved.